.github/workflows/rust.yml

```diff
@@ -37,7 +37,8 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings
+          # --all-targets makes it lint tests too
+          args: --all-targets -- --deny warnings
```

Comment:
Thanks for expanding `-D` to `--deny` here. Makes it easier to remember what each argument is doing. 👍